pkg/plugin/plugin.go

```diff
@@ -18,9 +18,49 @@ const (
 	MethodSendNotification = "SendNotification"
 )
 
+// FormElement describe a form element required for the channel config
+type FormElement struct {
+	// Element name
+	Name string `json:"name"`
+	// Element type: string = text, bool = checkbox, text = textarea, option = select, options = select[multiple], secret = password
+	Type string `json:"type"`
+	// Element label map. Locale in the standard format (language_REGION) as key and corresponding label as value.
+	//
+	// e.g. {"en_US": "Save", "de_DE": "Speichern"}
+	Label map[string]string `json:"label"`
+	// Element placeholder
+	Placeholder string `json:"placeholder,omitempty"`
+	// Element title
+	Help map[string]string `json:"help,omitempty"`
```

Comment:
Why is `Help` documented as title?

Reply:
This is based on Johannes' suggestions. Help is used as the title attribute of the element. When the user moves the mouse pointer over an element, a tooltip is displayed with given message.

Reply:
That implies that someone who reads "Element title" will think of the HTML title attribute. So that information would be much more helpful in the comment:

> When the user moves the mouse pointer over an element, a tooltip is displayed with given message.